create/serviceconnection.go

```diff
@@ -0,0 +1,213 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	"github.com/alecthomas/kong"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	networking "github.com/ninech/apis/networking/v1alpha1"
+	"github.com/ninech/nctl/api"
+)
+
+type serviceConnectionCmd struct {
```

Comment:
This implements the resource the same way as in the API, which makes sense. But from a Usability POV it might be nice to update an App or KVS instead of having to create a separate service connection:

```bash
nctl update app web-prod --service-connections kvs/sessions-prod
```

or

```bash
nctl update kvs sessions-prod --service-connections app/web-prod
```

What do you think?